docs/standard/modern-web-apps-azure-architecture/development-process-for-azure.md

```diff
@@ -0,0 +1,112 @@
+
+To get started with developing an ASP.NET Core application using CI/CD, you can use Visual Studio Team Services or your organization's own Team Foundation Server (TFS).
+
+### Initial Setup
+
+To create a release pipeline for your app, you need to have your application code in source control. Set up a local repository and connect it to a remote repository in a team project. Follow these instructions:
+
+-   [Share your code with Git and Visual Studio](https://www.visualstudio.com/en-us/docs/git/share-your-code-in-git-vs) or
+
+-   [Share your code with TFVC and Visual Studio](https://www.visualstudio.com/en-us/docs/tfvc/share-your-code-in-tfvc-vs)
+
+Create an Azure App Service where you'll deploy your application. Create a Web App by going to the App Services blade on the Azure portal. Click +Add, select the Web App template, click Create, and provide a name and other details. The web app will be accessible from {name}.azurewebsites.net.
+
+![AzureWebApp](./media/image10-2.png)
+
+**Figure 10-2.** Creating a new Azure App Service Web App in the Azure Portal.
+
+Your CI build process will perform an automated build whenever new code is committed to the project's source control repository. This gives you immediate feedback that the code builds (and, ideally, passes automated tests) and can potentially be deployed. This CI build will produce a web deploy package artifact and publish it for consumption by your CD process.
+
+[Define your CI build process](https://www.visualstudio.com/en-us/docs/build/apps/aspnet/aspnetcore-to-azure#ci)
+
+Be sure to enable continuous integration so the system will queue a build whenever someone on your team commits new code. Test the build and verify that it is producing a web deploy package as one of its artifacts.
+
+When a build succeeds, your CD process will deploy the results of your CI build to your Azure web app. To configure this, you create and configure a *Release*, which will deploy to your Azure App Service.
+
+[Define your CD release process](https://www.visualstudio.com/en-us/docs/build/apps/aspnet/aspnetcore-to-azure#cd)
+
+Once your CI/CD pipeline is configured, you can simply make updates to your web app and commit them to source control to have them deployed.
+
+### Workflow for developing Azure-hosted ASP.NET Core applications
+
+Once you have configured your Azure account and your CI/CD process, developing Azure-hosted ASP.NET Core applications is simple. The following are the basic steps you usually take when building an ASP.NET Core app, hosted in Azure App Service as a Web App, as illustrated in Figure 10-3.
+
+![EndToEndDevDeployWorkflow](./media/image10-3.png){width="6.25in" height="3.75in"}
```

Comment:
The style tag is showing in the review url:  https://review.docs.microsoft.com/en-us/dotnet/standard/modern-web-apps-azure-architecture/development-process-for-azure?branch=pr-en-us-3357

Reply:
this image text should say end-to-end

Reply:
both fixed.